src/test/javascript/javascript-parser_test.ts

```diff
@@ -33,12 +35,14 @@ suite('JavaScriptParser', () => {
   suite('parse()', () => {
 
     test('parses classes', () => {
+      // TODO(usergenic): I had to modify this test fixture because Babylon
+      // doesn't appreciate the keyword abuse of `const let = ...`.
```

Comment:
This TODO can be removed. `const let` is an early error it seems, so babel's doing the right thing here.